app-mobilephone/smstools/smstools-3.1.21-r2.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit systemd tmpfiles toolchain-funcs
+
+DESCRIPTION="Send and receive short messages through GSM modems"
+HOMEPAGE="http://smstools3.kekekasvi.com/"
+SRC_URI="http://smstools3.kekekasvi.com/packages/smstools3-${PV}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+IUSE="stats"
+
+DEPEND="
+	acct-group/sms
+	acct-user/smsd"
+RDEPEND="${DEPEND}
+	sys-process/procps
+	stats? ( dev-libs/mm )"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-gawk-location.patch
+	"${FILESDIR}"/${P}-fno-common.patch
+)
+
+S="${WORKDIR}"/${PN}3
+
+src_prepare() {
+	default
+	if use stats; then
+		sed -i -e "s:CFLAGS += -D NOSTATS:#CFLAGS += -D NOSTATS:" \
+			"${S}/src/Makefile" || die
+	fi
+	echo "CFLAGS += ${CFLAGS}" >> src/Makefile || die
+}
+
+src_compile() {
+	cd src || die
```

Comment:
ditch this, try adding `-C src` to the emake call

Reply:
Done